layers/llc.go

```diff
@@ -62,6 +62,9 @@ func decodeLLC(data []byte, p gopacket.PacketBuilder) error {
 	if l.DSAP == 0xAA && l.SSAP == 0xAA {
```

Comment:
might be better to make this a switch:

```go
switch {
  case l.DSAP == 0xAA && l.SSAP == 0xAA:
    ...
  case l.DSAP == 0x42 && l.SSAP == 0x42:
    ...
}
```

I expect we'll add things here later on, and a switch seems more useful for doing that.

Reply:
done